app.py

```diff
@@ -126,7 +126,7 @@ def register_proposal(caller_identity):
     node.ip = request.remote_addr
     node.country = detect_country(node.ip) or ''
     node.proposal = json.dumps(proposal)
-    node.updated_at = datetime.utcnow()
+    node.update_timestamp()
```

Comment:
Timestamp is too abstract, could be several dates.
`markNodeActive()`?

Reply:
Updated.